src/reducer/admin/index.js

```diff
@@ -1,12 +1,13 @@
 import { combineReducers } from 'redux';
-import resourceReducer from './resource';
+import { createSelector } from 'reselect';
+import resourceReducer, * as resourceSelectors from './resource';
```

Comment:
The `import *` served as a *namespace* for the selectors. There are a lot of name collisions between selectors of different levels.

Maybe collisions could be avoided by thoroughly renaming the selectors, but for the highest level selectors we could end up with names like these:
* `getAdminResourceRecord`
* `isAdminUISidebarOpen` :worried:
* `getAdminNotification`
* ...

Seems pretty straightforward for most and a pain for a few... would that be an issue for you?